c2corg_ui/static/partials/feed.html

```diff
@@ -1,76 +1,269 @@
+            </div>
+            <div class="grow">
+            </div>
+          </div>
+        </div>
+        <div class="parent-infop parent-infop-first" ng-if="::doc['document']['areas'] != null">
+          <a ng-href="{{::feedCtrl.createURLArea(doc['document']['areas'])}}" title="{{::feedCtrl.showArea(doc['document']['areas'])}}" ng-click="$event.stopPropagation()">
+            <div class="infop-bar">
+              <div class="parent-bubble">
+                <div class="bubble">
+                  <span class="glyphicon glyphicon-map-marker"></span>
+                </div>
+              </div>
+              <div class="text-bubble">
+                {{::feedCtrl.showArea(doc['document']['areas'])}}
+              </div>
+              <div class="grow">
+              </div>
+            </div></a>
+        </div>
+        <div class="parent-infop">
+          <div class="infop-bar">
+
+
+            <div class="parent-bubble" ng-repeat="activity in ::doc['document']['activities']">
+              <a href="#" class="shcom" uib-tooltip="{{activity | translate}}"> 
+                <div class="bubble">
+                  <span class="route-activity icon-{{::activity}}"></span>
+                </div>
+              </a>
+            </div>
+
+            <div class="grow">
+            </div>
+
+
+            <div class="parent-bubble">
+              <a href="#" class="link share-facebook" ng-if="::doc['document']['condition_rating']" uib-tooltip="{{'condition_rating' | translate}} : {{doc['document']['condition_rating'] | translate}}" tooltip-append-to-body="true">
+                <div class="condition-rating" ng-class="::doc['document']['condition_rating']" ></div>
+              </a>
+            </div>
+
+            <div class="parent-bubble"><div class="bubble bquality">
+              <a href="#" class="link share-facebook" uib-tooltip="{{doc['document']['quality'] | translate}}" tooltip-append-to-body="true">
+
+                <span class="quality-{{::doc['document']['quality']}}"></span>
+              </a></div>
+            </div>
+          </div>
+        </div>
+      </article>
+    </div> 
+    <div style="clear:both">&nbsp;</div>
+
+    <h4 class="text-center text-danger" translate ng-show="feedCtrl.error">Sorry, there was an error while getting the feed.</h4>
+    <h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading feed, please wait...</h4>
+    <h4 class="text-success text-center" translate ng-show="feedCtrl.feedEnd">No more feed to load.</h4>
+  </div>
+</div>
+<div class="feed-cr">
+  <div class="info">
+    <h3>Échos du forum</h3>
+    <h4 class="text-danger"translate ng-show="feedCtrl.errorForum">Sorry, there was an error while getting the forum data.</h4>
+    <div class="fpost" ng-repeat="post in feedCtrl.forums track by post.id" ng-if="::feedCtrl.forums.length > 0">
+      <a href="https://forum.camptocamp.org/t/{{::post['slug']}}/{{::post['id']}}" title="{{::post['title']}}">
```

Comment:
Can't you use the ``discourseUrl`` instead of hardcoding the prod forum base URL?